src/hooks/usePoolData.ts

```diff
@@ -248,14 +253,25 @@ export default function usePoolData(name?: string): PoolDataHookReturnType {
             value: priceDataForPool.underlyingTokenBalances1e18[i] || Zero,
             address: token.address,
           })) || []
-        const userPoolTokens = expandedPool.tokens.map((token, i) => ({
-          isOnTokenLists: token.isOnTokenLists,
-          symbol: token.symbol,
-          name: token.name,
-          decimals: token.decimals,
-          value: userPoolTokenBalances[i],
-          address: token.address,
-        }))
+        const userPoolTokens =
+          expandedPool.tokens.map((token, i) => ({
+            isOnTokenLists: token.isOnTokenLists,
+            symbol: token.symbol,
+            name: token.name,
+            decimals: token.decimals,
+            value: userPoolTokenBalances[i],
+            address: token.address,
+          })) || []
```

Comment:
Because ` expandedPool.underlyingTokens` is optional, I gave empty array rather than returning undefined.
But actually, expandedPool is always returning value so after clearing null value, we can remove this empty array.